assets/tvs/multitv/includes/multitv.class.php

```diff
@@ -359,7 +359,7 @@ function renderMultiTVFormElement($fieldType, $fieldName, $fieldElements, $field
                 $fieldType = 'date';
                 break;
             case 'image' :
-                if ($this->display == 'datatable' || $this->display == 'dbtatable' || $this->display == 'vertical') {
+                if ($this->display == 'datatable' || $this->display == 'dbtatable' || $this->display == 'vertical' || $this->display == 'horizontal') {
```

Comment:
This line is enough, the other lines lead to class names conflict i.e. if you name a field 'inline'.